services/web-app/components/nav-tree/nav-tree.module.scss

```diff
@@ -97,7 +97,15 @@
   .section-heading {
     padding-bottom: spacing.$spacing-03;
     border-bottom: 1px solid theme.$layer-accent;
-    margin: spacing.$spacing-08 spacing.$spacing-05 spacing.$spacing-03;
+    margin: 0 spacing.$spacing-05 spacing.$spacing-03;
     @include type.type-style('label-01');
   }
+
+  .section {
+    margin-top: spacing.$spacing-08;
+  }
+
+  li:last-child {
+    margin-bottom: spacing.$spacing-10;
+  }
```

Comment:
si! for short view ports

Reply:
I think we need to move this to the wrapping element instead, otherwise we are getting this 

![Screen Shot 2022-06-09 at 9 51 05 AM](https://user-images.githubusercontent.com/2753488/172876896-f96f3f5f-fa20-4bbb-82e6-da1ca9140ab9.png)